@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -707,11 +707,13 @@ export interface MarkerTechniqueParams extends BaseTechniqueParams {
      */
     poiNameField?: string;
     /**
-     * Name of [[ImageTexture]] definition to use.
+     * Name of [[ImageTexture]] definition or image in {@link @here/harp-mapview#userImageCache}
```

Comment:
I think the imageTextures from the theme go to some internal cache in MapViewThemeManager, not to MapView.userImageCache

Reply:
I meant, it's the name of either a theme's image or a user image in userImageCache. I'll rewrite the documentation to make it clearer.